ui/components/multichain/pages/send/components/quote-card/index.tsx

```diff
@@ -110,6 +110,16 @@ export function QuoteCard({ scrollRef }: QuoteCardProps) {
     }
 
     if (timeLeft <= 0) {
+      trackEvent(
```

Comment:
+1 seeing this locally

Lost MixPanel access again so can't check rn, but this might be the reason we're seeing low usage for Swap+Send

Reply:
looking at this a bit more closely it seems that the component transitively called `updateSendQuote()` through other functions like:

* `updateRecipient()`
* `updateSendAmount()`
* `updateSendAsset()`
* `updateSendHexData()`
* `toggleSendMaxMode()`

I'm thinking for most of these we can check if the `sourceAsset === destAsset` and determine at the point of those functions are dispatched if we need to fire a tracking event, but that feels a little excessive. I'd like to put it in the `updateSendQuote()` but react yells at me since it's not within a component. Do either of you have any ideas on how to clearly handle this?

Reply:
@micaelae came up with a great idea of using `useEffect()` and watching a particular variable to determine if we need to publish an event or not. Let me know if you like this approach.

Reply:
That approach makes sense to me!